scripts/extensions/auto-tagging-widget/src/types.ts

```diff
@@ -3,6 +3,8 @@ export interface ITagUi {
     description?: string;
     qcode: string;
     source?: string;
+    original_source?: string;
+    aliases?: Array<string>;
```

Comment:
Could you add comments on what those are?

Reply:
it's actually aliases :wink: so other names which are used for the concept